examples/dgfip_c/const.h

```diff
@@ -95,7 +95,8 @@ extern void free_erreur();
 #ifdef FLG_OPTIM_MIN_MAX
 
 #define my_floor(a)	(floor_g((a) + 0.000001))
-#define my_arr(a)	(floor_g((a) + 0.50005))
+//#define my_arr(a)	(floor_g((a) + 0.50005)) /* Ancienne version (2021) */
+#define my_arr(a)	(((a) < 0.0) ? ceil_g((a) - .50005) : floor_g((a) + .50005))
```

Comment:
This needs to be update here as well:
https://github.com/MLanguage/mlang/blob/9c95761d7b7795fdc8d2d6da0f30176027717bf9/src/mlang/backend_ir/bir_interpreter.ml#L109-L115